### PR TITLE
Fix remaining EVM tests

### PR DESCRIPTION
**Pull request by ayrat555**
- fix remaining EVM tests
- remove ex_rlp from umbrella app
- update ex_rlp to the latest version (0.3.0)
- fix test submodule

### CODE REVIEW

File: apps/evm/lib/evm/log_entry.ex [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+          255, 255>>
+      ]
+
+  """
+  @spec to_list(t()) :: [binary()]
+  def to_list(log) do
+    topics =
+      log.topics
+      |> Enum.map(fn topic ->
```
Comment (masonforest) [inline apps/evm/lib/evm/log_entry.ex]: would this work?

`Enum.map(&Helpers.left_pad_bytes/1)`

File: apps/evm/test/evm_test.exs [hunk 2]
```diff
@@ -301,4 +246,10 @@ defmodule EvmTest do
       {address, storage}
     end)
   end
+
+  def logs_hash(logs) do
+    logs
+    |> ExRLP.encode()
+    |> :keccakf1600.sha3_256()
```
Comment (masonforest) [inline apps/evm/test/evm_test.exs]: do we have access to `exeth_crypto` here?

File: apps/evm/lib/evm/log_entry.ex (hunk 1 above)
Comment (ayrat555) [reply apps/evm/lib/evm/log_entry.ex]: yes, thanks

File: apps/evm/test/evm_test.exs (hunk 2 above)
Comment (ayrat555) [reply apps/evm/test/evm_test.exs]: @masonforest no, in this project we do not  have access to `exeth_crypto`. I'll try to replace all direct usages of `:keccakf1600` to `exeth_crypto`  now

**Pull request merged**